src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -19,7 +19,7 @@ CORTXFS_FS_CLI="/usr/bin/cortxfscli"
 NFS_INITIALIZED=/var/lib/nfs/nfs_initialized
 NFS_SETUP_LOG=/var/log/nfs_setup.log
 DEFAULT_EXPORT_OPTION="proto=nfs,secType=sys,Filesystem_id=192.1,client=1,"
-DEFAULT_EXPORT_OPTION+="clients=*,Squash=no_root_squash,access_type=RW,protocols=4,"
+DEFAULT_EXPORT_OPTION+="clients=*,Squash=no_root_squash,access_type=RW,protocols=3,"
```

Comment:
We also need to add "Disable_ACL = true" parameter as default to be sure that the ACL code is not exercised.